styles/style.css

```diff
@@ -0,0 +1,152 @@
+body {
+    box-sizing: border-box;
+    background-color: #DCDCDC;
+}
+.header {
+    background-image: url("../images/header-image.jpg");
+    background-color: #003C63;
+    background-size: 100%;
+    height: 15rem;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: row;
+  }
+.caption {
+    display: block;
+    color: #42aa04;
+    font-style: italic;
+    line-height: 10%;
```

Comment:
try not to use percents on line-height

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height